crates/polars-lazy/src/frame/pivot.rs

```diff
@@ -57,8 +60,8 @@ where
 {
     // make sure that the root column is replaced
     let agg_expr = agg_expr.map(|agg_expr| {
-        let expr = prepare_eval_expr(agg_expr);
-        PivotAgg::Expr(Arc::new(PivotExpr(expr)))
+        // let expr = prepare_eval_expr(agg_expr); // TODO TBD: Not needed? Intent unclear?
```

Comment:
This isn't needed indeed iff the columns refer to the columns in the dataframe. Does `pl.element` do this?